lib/rubycritic/core/analysed_module.rb

```diff
@@ -27,6 +28,17 @@ def rating
       @rating ||= Rating.from_cost(cost)
     end
 
+    def lines
+      return @lines if @lines
+      output = `wc -l < "#{path}"`.to_i
+      result = $?.success?
+      if result
+        @lines ||= output
+      else
+        @lines ||= File.open(path).lines.count
+      end
+    end
```

Comment:
I confirmed that the ruby version fast faster (about half the time on my tests: medium sized ruby project on MBP running Yosemite) so I moved it to use ruby only.